commands/item.js

```diff
@@ -38,24 +40,60 @@ Examples:
 
   let temp = argv._;
   temp = temp.filter(x => x !== 'item');
-  const itemName = temp.join(' ');
+  let itemName = temp.join(' ');
+
+  let candidates = resolveItemName(itemName); // check item name against local static array 
```

Comment:
I guess this comes down to personal preference. I find it useful to have two levels for comments, one pertaining to all code beneath it (until next full-line comment, closure, paragraph...), one limited to the actual line. It's also a way to keep code from blowing up too much, and make sure you do not pass on commenting just because it breaks the code flow. I've changed it somewhat though in the next commit.